### PR TITLE
update function for checking exemptedDomains

### PR DESCRIPTION
**Pull request by martgil**
This PR will update the filtering process for exempted domains making it more precise.

Example: 
emails: [ 'something@domain.test', 'somehting@domain.bigtest' ]
comments found: [ 'somehting@domain.bigtest' ]

reference: https://github.com/martgil/test-repo4/runs/5984631390?check_suite_focus=true

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -6218,7 +6218,7 @@ try {
   const comment = payload.comment ? payload.comment.body : payload.issue.body;
   console.log(`comment: ${comment}`)
   const exemptDomainsInput = core.getInput('exemptions')
-  const exemptDomains = exemptDomainsInput.split(',').map(d => d.toLowerCase()).map(d => '@' + d);
+  const exemptDomains = exemptDomainsInput.split(',').map(d => d.toLowerCase()).map(d => d);
```
Comment (IvanPizhenko) [inline dist/index.js:6221]: Remove `.map(d => d)`

File: index.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ try {
   const comment = payload.comment ? payload.comment.body : payload.issue.body;
   console.log(`comment: ${comment}`)
   const exemptDomainsInput = core.getInput('exemptions')
-  const exemptDomains = exemptDomainsInput.split(',').map(d => d.toLowerCase()).map(d => '@' + d);
+  const exemptDomains = exemptDomainsInput.split(',').map(d => d.toLowerCase()).map(d => d);
```
Comment (IvanPizhenko) [inline index.js]: Remove `.map(d => d)`

**Review by IvanPizhenko — CHANGES_REQUESTED** [review]
Small change required

File: index.js (hunk 2 above)
Comment (martgil) [reply index.js]: thank you for reviewing this sir @IvanPizhenko . I just removed the unnecessary code.

**Comment by martgil** [comment]
sir @IvanPizhenko ready for review. Thank you!

**Review by IvanPizhenko — APPROVED** [review]

**Pull request merged**